waspc/src/Wasp/EntityPSL.hs

```diff
@@ -0,0 +1,10 @@
+-- TODO(matija): remove PSL suffix, added it to avoid clashes with the existing Entity module.
```

Comment:
Well ok, maybe we will have multiple ways to define Entities, so this is not so terrible actually, I like it.

Reply:
ok cool, might be nice